focus-ios/focus-ios-tests/XCUITest/FindInPageTest.swift

```diff
@@ -5,6 +5,8 @@
 import XCTest
 
 class FindInPageTest: BaseTestCase {
```

Comment:
Since these tests have been added to the Page Action Menu in Test suite, I would either rename this test suite or under the Page Action Menu in test reail, create a Find in Page section where we have these tests